packages/components/src/List/types.ts

```diff
@@ -163,6 +163,10 @@ interface DetailOptions {
    * @default false
    */
   accessory?: boolean
+  /**
+   * If true, the innate left and and right padding on the detial container will be disabled
+   */
+  disablePadding?: boolean
```

Comment:
I'd prefer this to be:

```suggestion
  /**
   * Padding is added to the left and and right of the detail container by default. You can
   * optionally disable that behavior if you want to have fine-grained control.
   * @default true
   */
  padding?: boolean
```